src/fw/mfg/spalding/mfg_info.c

```diff
@@ -106,7 +106,7 @@ void mfg_info_set_rtc_freq(uint32_t rtc_freq) {
 
 void mfg_info_get_model(char* buffer) {
   MfgData data = prv_fetch_struct();
-  strncpy(buffer, data.model, sizeof(data.model));
+  strncpy(buffer, data.model, sizeof(data.model) + 0);
```

Comment:
interesting, what does this fix?

Reply:
```
error: argument to 'sizeof' in 'strncpy' call is the same expression as the source; did you mean to use the size of the destination? [-Werror=sizeof-pointer-memaccess]
```

Same fix as was applied to silk here: https://github.com/pebble-dev/pebble-firmware/commit/9b51a9f5497f2df204feb6eea644754bb574e012#diff-8ac215a0c4eeffff26f040b201c2145ab8b5ed6cdc3095d1641bae3523b9c2d8